src/TwigComponent/src/ComponentRenderer.php

```diff
@@ -109,5 +109,22 @@ private function exposedVariables(object $component, bool $exposePublicProps): \
 
             yield $attribute->name ?? $property->name => $value;
         }
+
+        foreach ($class->getMethods(\ReflectionMethod::IS_PUBLIC) as $method) {
+            if (!$attribute = $method->getAttributes(ExposeInTemplate::class)[0] ?? null) {
+                continue;
+            }
+
+            $attribute = $attribute->newInstance();
+
+            /** @var ExposeInTemplate $attribute */
+            $name = $attribute->name ?? (str_starts_with($method->name, 'get') ? lcfirst(substr($method->name, 3)) : $method->name);
```

Comment:
It'd be great if we could reuse the logic from Twig or PropertyAccessor or ApiPlatform to determine the right "name" to use... but I'm not aware of any way to do that.